components/EditClothesAmountButton.jsx

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from 'react';
+import Icon from "react-native-vector-icons/AntDesign";
+import { deleteClothesFromBasket, patchClothesCount } from '../utils/api';
+import {
+  View,
+  StyleSheet,
+  Text,
+} from 'react-native';
+
+const EditClothesAmountButton = ({ basket, setBasket, item, setTotalAmount }) => {
+  const [amount, setAmount] = useState(item?.basket_count ?? 0);
+  const [itemPrice, setItemPrice] = useState(parseFloat(item?.price?.substr(1)) ?? 0);
+
```

Comment:
is this because its coming from the database as a string?

Reply:
yep, price is a string which has a dollar sign as well

Reply:
oof my bad, last time i use money in a db